stl/inc/chrono

```diff
@@ -4608,6 +4608,22 @@ namespace chrono {
         return _Istr;
     }
 
+    namespace _From_stream_adl_only {
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, VSO-1681199
+        void from_stream() = delete; // Block unqualified name lookup
+#else // ^^^ no workaround / workaround vvv
+        void from_stream();
+#endif // ^^^ workaorund ^^^
+
+        template <class _Parsable, class _CharT, class _Traits, class... _Rest>
+        concept _Can_from_stream = requires(
+            basic_istream<_CharT, _Traits>& __istr, const _CharT* __s, _Parsable& __parsed, _Rest&&... __rest_args) {
+            from_stream(__istr, +__s, __parsed, _STD forward<_Rest>(__rest_args)...); // intentional ADL
+        }

```

Comment:
There is a missing semicolon here, causing catastrophic build failure:

```
C:\a\_work\1\s\stl\inc\chrono(4623,5): error C2143: syntax error: missing ';' before '}'
    } // namespace _From_stream_adl_only
    ^
```

As a reminder, please build the STL and run affected tests before pushing a PR.

Reply:
Oh, I must have saved a wrong version. 😿

Reply:
Ctrl-S, humanity's eternal nemesis :floppy_disk: :joy_cat: